# OpenStax Chemistry, 2nd Ed. &ndash; Dean's Errata Submissions
## Chapter 5

**Execute this cell to display chemical formulas.**
It contains a LaTeX require for mhchem, which even applies to later cells.
$ \require{mhchem} $

### Chapter 5 - Formulas and Concepts

#### Chapter 5.1

$$ q_{substance\ a} + q_{substance\ b} = 0 $$
$$ \text{specific heat}\  c = \frac{q}{m\Delta T} $$
$$ \Delta T = T_{final} - T_{initial} = T_f - T_i $$
$$
\begin{align}
q &= c \cdot m \cdot \Delta T \\ 
&= c \cdot m \cdot (T_{final} - T_{initial}) \\
&= m c \Delta T \quad \text{Can be pronounced m-cat}
\end{align}
$$
$$ (c_a \cdot m_a \cdot \Delta T_a) + (c_b \cdot m_b \cdot \Delta T_b) = 0 $$
$$ (c_a \cdot m_a \cdot (T_{a\ final} - T_{a\ initial})) + (c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})) = 0 $$
$$ (c_a \cdot m_a \cdot (T_{a\ final} - T_{a\ initial})) = - (c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})) $$

#### Chapter 5.3

$$ \begin{align}
q &:= \text{heat (heat flow into system is positive)} \\
q_p &:= \text{heat under constant P} \\
w &:= \text{work (work done on system is positive, by system is negative)} \\
U &:= \text{internal energy} (\equiv E) \\
P &:= \text{pressure} \\
V &:= \text{volume} \\
H &:= \text{enthalpy} \\
\\
\Delta U &= q + w \\
H &= U + PV \text{ (unmeasurable)} \\
\Delta H &= \Delta U + \Delta (PV) \text{ (measurable, negative for exothermic)} \\
&= q + w + \Delta (PV) \\
P \Delta V &= -w \\
\Delta H &= \Delta U + P \Delta V \text{ (assuming constant P)} \\
&= q_p + w + - w \\
&= q_p \\
\\
1\text{ bar} &= 0.987\text{ atm} \\
\Delta H\degree &:= \text{Enthalpy change under standard conditions/standard state.} \\
&\qquad\text{Book uses 1 atm and 298.15 K (25 $\degree$C), slightly different than IUPAC.} \\
&\qquad\text{For solutions the standard conditions are 1 M molarity instead of 1 atm.} \\
\Delta H\rlap\degree_c &:= \text{Standard enthalpy of combustion of 1 mol under standard conditions} \\
\Delta H\rlap\degree_f &:= \text{Standard enthalpy of formation of 1 mol under standard conditions} \\
&\qquad\text{This is formation of 1 mol of a substance from free elements.} \\
\end{align} $$

**Hess’s law:** If a process can be written as the sum of several stepwise processes, the enthalpy change of the total process equals the sum of the enthalpy changes of the various steps.

If $n$ are the stoichiometric coefficients, then...
$$ \begin{align}
\Delta H \rlap \degree _{reaction} &=
        \sum{n \cdot \Delta H \rlap \degree _f (\text{products})}
        - \sum{n \cdot \Delta H \rlap \degree _f (\text{reactants})}
\end{align} $$


### Chapter 5 &ndash; Examples

#### Example 5.3 Solution

Define $a$ to be the metal, and $b$ to be the water.
$$ T_{a\ final} - T_{a\ initial} = - \left(\frac{c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})}{c_a \cdot m_a}\right) $$
$$ T_{a\ initial} = \left(\frac{c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})}{c_a \cdot m_a}\right) + T_{a\ final} $$
$$ m_b = 425 g $$
$$ c_b = 4.184 \frac{J}{g {\degree}C} $$
$$ T_{b\ final} = 42.7 {\degree}C $$
$$ T_{b\ initial} = 24.0 {\degree}C $$
$$ m_a = 360.0 g $$
$$ c_a = 0.449 \frac{J}{g {\degree}C} $$
$$ T_{a\ final} = T_{b\ final} = 42.7 {\degree}C $$
$$ T_{a\ initial} = \left(\frac{425 g \cdot 4.184 \frac{J}{g {\degree}C} \cdot (42.7 {\degree}C - 24.0 {\degree}C)}{360.0 g \cdot 0.449 \frac{J}{g {\degree}C}}\right) + 42.7 {\degree}C $$

In [1]:
((425 * 4.184 * (42.7 - 24.0)) / (360.0 * 0.449)) + 42.7

248.41851026973524

$$ T_{a\ initial} = 248 {\degree}C $$

#### Example 5.3 Check Your Learning 1

$$ T_{a\ initial} = \left(\frac{c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})}{c_a \cdot m_a}\right) + T_{a\ final} $$

In [2]:
((4.184 * 390 * (39.9 - 22.6)) / (0.385 * 248))

295.6582320904901

In [3]:
((4.184 * 390 * (39.9 - 22.6)) / (0.385 * 248)) + 39.9

335.5582320904901

336 degrees C

#### Example 5.3 Check Your Learning 2

$$ (c_a \cdot m_a \cdot (T_{a\ final} - T_{a\ initial})) = - (c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})) $$
Define $a$ to be the metal, and $b$ to be the water.
Note that $T_{a\ final}$ should always equal $T_{b\ final}$.
$$ (c_a \cdot m_a \cdot (T_{final} - T_{a\ initial})) = - (c_b \cdot m_b \cdot (T_{final} - T_{b\ initial})) $$
$$ \require{cancel} $$
$$ \frac{\cancel{c_a \cdot m_a \cdot} (T_{final} - T_{a\ initial})}{\cancel{c_a \cdot m_a}(c_b \cdot m_b)} = - \frac{\cancel{c_b \cdot m_b \cdot} (T_{final} - T_{b\ initial})}{(c_a \cdot m_a)\cancel{c_b \cdot m_b}} $$
$$ \frac{T_{final} - T_{a\ initial}}{c_b \cdot m_b} = - \frac{T_{final} - T_{b\ initial}}{c_a \cdot m_a} $$
$$ \frac{T_{final}}{c_b \cdot m_b} - \frac{T_{a\ initial}}{c_b \cdot m_b} = - \frac{T_{final}}{c_a \cdot m_a} - \frac{T_{b\ initial}}{c_a \cdot m_a}$$

**Wait, back up**
$$
\begin{align}
c_a \cdot m_a \cdot (T_{final} - T_{a\ initial}) &=
- c_b \cdot m_b \cdot (T_{final} - T_{b\ initial})
\\[1 ex]
% c_a \cdot m_a \cdot T_{final} -
% c_a \cdot m_a \cdot T_{a\ initial} &=
% - c_b \cdot m_b \cdot T_{final} +
% c_b \cdot m_b \cdot T_{b\ initial}
% \\[1 ex]
c_a \cdot m_a \cdot T_{final} -
c_a \cdot m_a \cdot T_{a\ initial} &=
c_b \cdot m_b \cdot T_{b\ initial} -
c_b \cdot m_b \cdot T_{final}
\\[1 ex]
c_a \cdot m_a \cdot T_{final} +
c_b \cdot m_b \cdot T_{final} &=
c_b \cdot m_b \cdot T_{b\ initial} +
c_a \cdot m_a \cdot T_{a\ initial}
\\[1 ex]
(c_a \cdot m_a + c_b \cdot m_b) \cdot T_{final} &=
c_b \cdot m_b \cdot T_{b\ initial} +
c_a \cdot m_a \cdot T_{a\ initial}
\\[1 ex]
T_{final} &=
\frac{c_b \cdot m_b \cdot T_{b\ initial} + c_a \cdot m_a \cdot T_{a\ initial}}
     {c_a \cdot m_a + c_b \cdot m_b}
\end{align}
$$

Defining $a$ to be the copper and $b$ to be the water, we have...
$$
T_{final} =
\frac{4.184 J/g {\degree}C \cdot 390 g \cdot 22.6 {\degree}C +
      0.385 J/g {\degree}C \cdot 248 g \cdot 314 {\degree}C}
     {0.385 J/g {\degree}C \cdot 248 g + 4.184 J/g {\degree}C \cdot 390 g}
$$

In [4]:
((4.184 * 390 * 22.6) + (0.385 * 248 * 314)) / ((0.385 * 248) + (4.184 * 390))

38.70828373590237

38.7 degrees C

#### Example 5.4 Solution

$$ \begin{align}
\Delta T = T_{final} - T_{initial} &= T_f - T_i
\\[1ex]
c_{metal} \times m_{metal} \times \Delta T_{metal} &= -
c_{water} \times m_{water} \times \Delta T_{water}
\\[1 ex]
c_{metal} &= - \frac{c_{water} \times m_{water} \times \Delta T_{water}}
                    {m_{metal} \times \Delta T_{metal}}
\\[1 ex]
c_{metal} &= - \frac{4.184 J/g \degree C \times 60.0 g \times (28.5 - 22.0 \degree C)}
                    {59.7 g \times (28.5 - 100.0 \degree C)}
\end{align} $$

In [5]:
- (4.184 * 60.0 * (28.5 - 22.0)) / (59.7 * (28.5 - 100.0))

0.3822750114207401

Specific heat $c_{metal} = 0.382 J / g \degree C$, so the metal is probably Copper.

**<span style="color: red">Note:</span>** The book said 0.38 (and copper).  They had an interim step where (28.5 - 22.0) was reduced to (6.5).  I am guessing that explains why they rounded the answer to 2 significant digits.  I'm guessing that the rounding is arguable either way, because if you do the calculation all in one step you would round to 3 significant digits.

#### Example 5.4 Check Your Learning

In [6]:
- (4.184 * 75.0 * (29.7 - 24.0)) / (92.9 * (29.7 - 178.0))

0.12982876620355413

Specific heat $c_{metal} = 0.130 J / g \degree C$ for the silvery-gray metal.

The metal is probably lead, because the only other metal listed in Table 5.1 is gold, which does not have a silvery gray color.

<strong style="color: red">Note:</strong>  The book said 0.13 (and lead).  I am guessing resolving the (29.7 - 24.0) term resulted in rounding to 2 significant digits.

#### Example 5.5 Solution

The book gives -2.9 kJ as the answer but assumes:
- The density of the solutions are approximately the same as water, 1.0 g/mL
- The specific heat of the solutions are approximately the same as water 4.184 J / g &deg;C
- The calorimeter allows no heat to enter or leave.
I want to try removing the first two assumptions, and see if I can calculate a more specific answer.  I will assume that the solutions are not being added to any additional water.

So, the problem statement from the book is:

> When 50.0 mL of 1.00 M HCl(aq) and 50.0 mL of 1.00 M NaOH(aq), both at 22.0 &deg;C, are added to a coffee cup calorimeter, the temperature of the mixture reaches a maximum of 28.9 &deg;C. What is the approximate amount of heat produced by this reaction?

> $$ \ce{HCl(aq) + NaOH(aq) -> NaCl(aq) + H2O(l)} $$

$$ 50.0\ mL * 1.00\ mol/L = .0500 L * 1.00 mol/L = 0.500 mol HCl $$

<strong style="color:red">Wait.</strong> I do not know the density of aqueous solutions of HCl (a gas) or NaOH, nor do I know a good place to look them up, or calculate the densities at a particular molarity $M$.  I thought I could maybe calculate from the molar masses, but that does not help me with the densities.  Therefore, **nevermind**.


#### Example 5.5 Check Your Learning

When 100 mL of 0.200 M NaCl(aq) and 100 mL of 0.200 M AgNO<sub>3</sub>(aq), both at 21.9 °C, are mixed in a coffee cup calorimeter, the temperature increases to 23.5 °C as solid AgCl forms. How much heat is produced by this precipitation reaction? What assumptions did you make to determine your value? 

In [7]:
-((100 + 100) * 1.0 * 4.184 * (23.5 - 21.9))

-1338.8800000000012

$$ q_{reaction} = - q_{solution} = -1.3 kJ $$
We assumed that both input solutes were roughly the same density and specific heat as liquid water, and that the calorimeter did not absorb any heat or let any heat enter or escape.

**<span style="color:red">Note:</span>** The book said 1.34 kJ, which differs from the rounding behavior used in the Example 5.5 Solution just above the Check Your Learning, so I will report it.

#### Example 5.6 Solution

In [8]:
-((3.21 + 50.0) * 4.184 * (20.3 - 24.9))

1024.1009439999996

$$ \begin{align}
q_{rxn} = - q_{soln} &= - (3.21 g\ \ce{NH4NO3} + 50.0 g\ \ce{H2O}) * 4.184 \frac{J}{g \degree C} * (20.3 \degree C - 24.9 \degree C) \\
&= -(-1.02 kJ) \\[1 ex]
&= 1.02kJ 
\end{align}$$

The positive sign for $q_{reaction}$ means that heat has to be added to the reactants from the environment, making the reaction endothermic.  This assumes that the calorimeter does not provide or absorb heat or allow heat exchange with the outside environment.

<strong style="color:red">Note:</strong> The book gave +1.0 kJ as the answer, but I think that is because they incorporated 20.3 - 24.9 = -4.6 first into the rounding.  I also forgot to mention the assumption that the final solution's specific heat is approximately the same as that of water.  (I also fat fingered the specific heat of water at first, but fixed that when I saw my answer disagreed greatly with the book answer.)

#### Example 5.6 Check Your Learning

In [9]:
-((300 + 3.00) * 4.184 * (-1.05))

1331.1396

#### Example 5.7 Solution

The problem statement is:
> When 3.12 g of glucose, $\ce{C6H12O6}$, is burned in a bomb calorimeter, the temperature of the calorimeter increases from 23.8 °C to 35.6 °C. The calorimeter contains 775 g of water, and the bomb itself has a heat capacity of 893 J/°C. How much heat was produced by the combustion of the glucose sample?

First, let's determine the heat capacity $C_{sys}$ of the system as a whole.
$$ \begin{align}
C_{sys} &= C_{bomb} + C_{water} \\
&= C_{bomb} + m_{water} \times c_{water} \\
&= 893 J/\degree C + 775 g \times 4.184 J/g \degree C 
\end{align}$$
Then multiply by temperature change $\Delta T$ to get the heat energy produced $q_{rel}$.

$$ \begin{align}
q_{rel} &= -q_{rxn} \\
&= C_{sys} \times \Delta T \\
&= (C_{bomb} + m_{water} \times c_{water}) \times (T_f - T_i) \\
&= (893 J/\degree C + 775 g \times 4.184 J/g \degree C)
    \times (35.6 \degree C - 23.8 \degree C)
\end{align} $$

In [10]:
(893 + 775 * 4.184) * (35.6 - 23.8)

48800.08000000001

The solution I get is
$$ q_{rel} = 48.8 \times 10^3 J = 48.4 kJ $$
The book did it in a little different order.
$$ \begin{align}
q_{rxn} &= -(q_{water} + q_{bomb}) \\
&= -[4.184 \times 775 \times (35.6 - 23.8) + 893 \times (35.6 - 23.8)] \\
&= -48.8 kJ
\end{align} $$

In [11]:
-(4.184 * 775 * (35.6 - 23.8) + 893 * (35.6 - 23.8))

-48800.08

#### Example 5.7 Check Your Learning

In [12]:
(784 + 925 * 1 * 4.184) * 8.39

39048.73800000001

$$ q_{rxn} = -39.0 kJ $$

#### Example 5.8 Solution

In [13]:
2.9 / 0.0500

57.99999999999999

$ \text{The \textbf{thermochemical equation} is} $
$$ \ce{HCl(aq) + NaOH(aq) -> NaCl(aq) + H2O(l)}\qquad \Delta H = -58 kJ $$

#### Example 5.8 Check Your Learning

In [14]:
from dp import *
from decimal import *
display('Zn amus: ' + str(b.Zn))
mlzn = Decimal(1.34) / b.Zn
display('mlzn:    ' + str(mlzn))
display('dh 1mol: ' + str(Decimal('3.14') / mlzn))
display('---')
mlhcl = Decimal(60.0) / 10**3 * Decimal(0.750)
display('mlhcl:   ' + str(mlhcl))
display('dh 2mol: ' + str(Decimal('3.14') / mlhcl * 2))

'Zn amus: 65.38'

'mlzn:    0.02049556439278066809324040644'

'dh 1mol: 153.2038805970149162339445892'

'---'

'mlhcl:   0.0450'

'dh 2mol: 139.5555555555555555555555556'

The problem statement is
> When 1.34 g Zn(s) reacts with 60.0 mL of 0.750 M HCl(aq), 3.14 kJ of heat are produced. Determine the enthalpy change per mole of zinc reacting for the reaction:

$$ \ce{Zn(s) + 2HCl(aq) -> ZnCl2(aq) + H2(g)} $$
$$ \begin{align}
ml_{Zn} :&= \text{moles Zn tested} \\[1ex]
    &= 1.34\text{ g Zn} \cdot
        \frac{1\text{ mol Zn}}{65.38 \text{ g Zn}} \\[1ex]
    &= 0.020496\text{ mol Zn} \\[1ex]
\frac{\Delta H}{\text{1 mol Zn}} &= \frac{3.14\text{ kJ}}{ml_{Zn}} \\[1ex]
    &= 153\text{ kJ} \\[1ex]
ml_{HCl} :&= \text{moles HCl tested} \\[1ex]
    &= 60.0\text{ mL} \cdot
        \frac{1\text{ L}}{10^3\text{ mL}} \cdot
        \frac{0.750\ \text{mol}\ \ce{HCl(aq)}}{1\text{ L}} \\[1ex]
    &= 0.0450\text{ mol HCl (why not a multiple of $ml_{Zn}$?)} \\[1ex]
\frac{\Delta H}{\text{2 mol HCl}} &=
    \frac{3.14\text{ kJ}}{ml_{HCl}} \cdot
    2\text{ mol} \\[1ex]
&= 26.2\text{ kJ}
\end{align} $$

<strong style="color:red">Note:</strong> The book answer is $\Delta H = -153 \text{ kJ}$.  I don't know why calculating it from the HCl did not give the same answer as from Zn.

#### Example 5.9 Solution

In [15]:
def example5_9_sol(s):
    with localcontext(prec = 7):
        s_amu = s.C*12 + s.H*22 + s.O*11
        k_amu = s.K + s.Cl + s.O*3
        return {
            'Amus sucrose': s_amu,
            'Moles sucrose': Decimal('2.67') / s_amu,
            'Amus KClO3': k_amu,
            'Moles KClO3': Decimal('7.19') / k_amu,
            'Moles 8 KClO3': Decimal('7.19') / k_amu / 8,
            'Delta H': Decimal('43.7') / (Decimal('7.19') / k_amu / 8)
        }
example5_9_sol(b)
    

{'Amus sucrose': Decimal('342.296'),
 'Moles sucrose': Decimal('0.007800266'),
 'Amus KClO3': Decimal('122.55'),
 'Moles KClO3': Decimal('0.05866993'),
 'Moles 8 KClO3': Decimal('0.007333741'),
 'Delta H': Decimal('5958.760')}

Problem statement:
> A gummy bear contains 2.67 g sucrose, C12H22O11. When it reacts with 7.19 g potassium chlorate, KClO3, 43.7 kJ of heat are produced. Write a thermochemical equation for the reaction of one mole of sucrose:
> $$ \ce{C12⁢H22⁢O11⁡(aq) + 8⁢KClO3⁢(aq)⟶12⁢CO2⁢(g) +11⁢H2O(𝑙) +8⁢KCl⁡(aq)} $$

The book just said
> Be sure to take both stoichiometry and limiting reactants into account...

$$ \begin{align}
\text{Moles }\ce{C12H22O11} &= 2.67 g \cdot \frac{1 mol}{342.296 g} \\
    &= 0.007800\ mol \\
\text{Moles }\ce{KClO3} &= 7.19 g \cdot \frac{1 mol}{122.55 g} \\
    &= 0.05867\ mol \\
\frac{0.05867\ mol}{8} &= 0.007334
\end{align} $$

$\ce{KClO3}$ is the limiting reactant.  Although 0.007800 Moles of sucrose are available to react, only 0.007334 Moles actually reacted due to the limited amount of potassium chlorate.

$$ \begin{align}
\Delta H &= - \frac{43.7\ kJ}{0.007334\ mol} \\[1 ex]
    &= 5.96 \times 10^3\ kJ = 5.96 \times 10^6\ J
\end{align} $$

$$ \ce{C12⁢H22⁢O11⁡(aq) + 8⁢KClO3⁢(aq)⟶12⁢CO2⁢(g) +11⁢H2O(𝑙) +8⁢KCl⁡(aq)} \quad \Delta H = - 5.96 \times 10^6\ J$$


#### Example 5.9 Check Your Learning

In [16]:
def example5_9_check(s):
    with localcontext(prec = 7):
        amu_fe = s.Fe
        mol_fe = Decimal('1.42') / amu_fe
        mol_fe_round = round(mol_fe, 4)
        amu_cl = s.Cl*2
        mol_cl = Decimal('1.80') / amu_cl
        mol_cl_round = round(mol_cl, 4)
        return {
            'Fe amu': amu_fe,
            'Cl2 amu': amu_cl,
            'Fe mol ': mol_fe,
            'Cl2 mol': mol_cl,
            'Fe round': mol_fe_round,
            'Cl2 round': mol_cl_round,
            'Delta H': Decimal(8.60) / mol_cl,
            'Delta H Fe': Decimal(8.60) / mol_fe,
            'Delta H round': Decimal(8.60) / mol_cl_round,
            'Delta H round Fe': Decimal(8.60) / mol_fe_round,
        }
example5_9_check(b)

{'Fe amu': Decimal('55.85'),
 'Cl2 amu': Decimal('70.90'),
 'Fe mol ': Decimal('0.02542525'),
 'Cl2 mol': Decimal('0.02538787'),
 'Fe round': Decimal('0.0254'),
 'Cl2 round': Decimal('0.0254'),
 'Delta H': Decimal('338.7444'),
 'Delta H Fe': Decimal('338.2464'),
 'Delta H round': Decimal('338.5827'),
 'Delta H round Fe': Decimal('338.5827')}

In [17]:
# from IPython.display import display, HTML
sources = (b, i, c, n)
import pandas as pd
pd.set_option("display.float_format", str)
def compares(f):
    return pd.DataFrame({
        'b': f(b),
        'i': f(i),
        'n': f(n),
        'c': f(c)})

In [18]:
compares(example5_9_check)

,b,i,n,c
Fe amu,55.85,55.845,55.84,55.845
Cl2 amu,70.90,70.90,70.90,70.90
Fe mol,0.02542525,0.02542752,0.02542980,0.02542752
Cl2 mol,0.02538787,0.02538787,0.02538787,0.02538787
Fe round,0.0254,0.0254,0.0254,0.0254
Cl2 round,0.0254,0.0254,0.0254,0.0254
Delta H,338.7444,338.7444,338.7444,338.7444
Delta H Fe,338.2464,338.2162,338.1859,338.2162
Delta H round,338.5827,338.5827,338.5827,338.5827
Delta H round Fe,338.5827,338.5827,338.5827,338.5827


Problem statement:
> When 1.42 g of iron reacts with 1.80 g of chlorine, 3.22 g of FeCl2(s) and 8.60 kJ of heat is produced. What is the enthalpy change for the reaction when 1 mole of FeCl2(s) is produced?

First, write a balanced equation:
$$ \ce{Fe(s) + Cl2(g) -> FeCl2(s)} $$

In the code section above, we determine that $\ce{Cl2}$ is the limiting reactant.  Therefore:
$$ \begin{align}
\Delta H &= - \frac{8.60\ kJ}{0.02539\ mol} \\[1ex]
    &= - 339\ kJ
\end{align} $$

<strong style="color:red">Note: </strong> The book answer is -338 kJ, which I can only account for using Fe as the limiting reactant, which every atomic weight source I tried agreed is wrong.  I cannot account for the different answer with different atomic weight sources.  Nor can I account for the different answer with rounding.

**TODO: ### Re-examine, and if I cannot get the book answer, report it as a possible issue.**

#### Example 5.10 Solution

In [19]:
def example5_10_sol(s):
    with localcontext(prec = 7):
        io_amu = s.C*8 + s.H*18
        io_amu_round = round(io_amu, 2)
        return {
            'amu isooctane': io_amu,
            'delta h': 1 * 10**3 * Decimal('0.692') / io_amu * Decimal('-5465.5'),
            'delta h -5460': 1 * 10**3 * Decimal('0.692') / io_amu * Decimal('-5460'),
            'amu rounded': io_amu_round,
            'delta h rounded': 1 * 10**3 * Decimal('0.692') / io_amu_round * Decimal('-5465.5'),
            'delta h -5460 rounded': 1 * 10**3 * Decimal('0.692') / io_amu_round * Decimal('-5460'),
            'mol isooctane': 1 * 10**3 * Decimal('0.692') / io_amu,
            'mol isooctane rounded': 1 * 10**3 * Decimal('0.692') / io_amu_round,
            'bad intermediate moles': Decimal('6.07') * Decimal('-5465.5'),
            'bad intermediate moles -5460': Decimal('6.07') * Decimal('-5460'),
        }
compares(example5_10_sol)

,b,i,n,c
amu isooctane,114.224,114.2320,114.2320,114.232
delta h,-33111.48,-33109.16,-33109.16,-33109.16
delta h -5460,-33078.16,-33075.84,-33075.84,-33075.84
amu rounded,114.22,114.23,114.23,114.23
delta h rounded,-33112.64,-33109.74,-33109.74,-33109.74
delta h -5460 rounded,-33079.32,-33076.42,-33076.42,-33076.42
mol isooctane,6.058271,6.057847,6.057847,6.057847
mol isooctane rounded,6.058484,6.057953,6.057953,6.057953
bad intermediate moles,-33175.58,-33175.58,-33175.58,-33175.58
bad intermediate moles -5460,-33142.20,-33142.20,-33142.20,-33142.20


The problem statement is:
> Let us determine the approximate amount of heat produced by burning 1.00 L of gasoline, assuming the enthalpy of combustion of gasoline is the same as that of isooctane, a common component of gasoline. The density of isooctane is 0.692 g/mL.

From Table 5.2, isooctane's enthalpy of combustion is –5465.5 kJ/mol and the formula for isooctane is $\ce{C8H18}$.

$$ \begin{align}
\Delta H\rlap\degree_{c\ isooctane} &= –5465.5\ kJ/mol \\
\Delta H &= 1.00\ L
        \cdot \frac{10^3\ mL}{1\ L}
        \cdot \frac{0.692\ g}{1\ mL}
        \cdot \frac{1\ mol}{114.224\ g}
        \cdot \frac{-5465.5\ kJ}{1\ mol}
        \\
    &= -3.31 \times 10^4\ kJ = -33100\ kJ \\
\text{Heat produced} &= -\Delta H \\
    &= 33100\ kJ
\end{align} $$

<strong style="color:red">Note:</strong> The book gets two things wrong.
- It uses an incorrect enthalpy of combustion for isooctane which does not match Table 5.2 as claimed.
- It has a wrong intermediate result for the number of moles of isooctane.
If you fix both issues, the final answer, when rounded, is unchanged.


#### Example 5.10 Check Your Learning

In [20]:
def example5_10_check(s):
    with localcontext(prec = 7):
        amu_ac = s.C*2 + s.H*2
        return {
            'amu acetylene': amu_ac,
            'Delta H': Decimal('125') / amu_ac * Decimal('-1301.1'),
        }
example5_10_check(b)

{'amu acetylene': Decimal('26.036'), 'Delta H': Decimal('-6246.640')}

Problem statement:
> How much heat is produced by the combustion of 125 g of acetylene?

From Table 5.2 the formula for Acetylene is $\ce{C2H2(g)}$, and its enthalpy of combustion is -1301.1 kJ/mol.

$$ \begin{align}
\Delta H \rlap{\degree} _{c\ acetylene} &= -1301.1\ kJ/mol \\
\Delta H &= 125\ g\ \ce{C2H2}
        \cdot \frac{1\ mol\ \ce{C2H2}}{26.036\ g}
        \cdot \frac{-1301.1\ kJ}{1\ mol\ \ce{C2H2}} \\
    &= -6250\ kJ \\
\text{Heat produced} &= - \Delta H \\
    &= 6250\ kJ \\
\end{align} $$


#### Example 5.11 Solution

In [21]:
+286 / 2

143.0

$$ \Delta H \rlap \degree _f [\ce{O3(g)}]= + 143\ kJ/mol$$

#### Example 5.11 Check Your Learning

In [22]:
-92.3 * 2

-184.6

$$ \Delta H = -184.6\ kJ $$
**Oops:** I divided by 2 instead of multiplied at first.  My bad.

#### Example 5.12 Solution

Problem statement:
> Write the heat of formation reaction equations for:
> 
> (a) $\ce{C2H5OH(l)}$
> 
> (b) $\ce{Ca3(PO4)2(s)}$

$ \ce{2C(s, graphite) + 3H2(g) + \frac12O2(g) -> C2H5OH(l)} $

$ \ce{3Ca(s) + \frac12P4(s) + 4O2(g) -> Ca3(PO4)2(s)} $

**Note:** I didn't know I could add ", graphite" to the solid indicator for carbon until I saw the book answer.  Also, the 0 enthalpy of formation entries in Appendix G were useful for knowning the stable pure states of elements.

#### Example 5.12 Check Your Learning

Problem statement:
> Write the heat of formation reaction equations for:
>
> (a) $\ce{C2H5OC2H5(l)}$
>
> (b) $\ce{Na2CO3(s)}$

$\ce{4C(s, graphite) + 5H2(g) + \frac12 O2(g) -> C2H5OC2H5(l)}$

$\ce{Na(s) + C(s, graphite) + \frac32 O2(g) -> Na2CO3(s)}$

**Oops:** Put 2 Carbon atoms as input instead of one.

#### Example 5.13 Solution

In [23]:
-341.8 + - 57.7

-399.5

#### Example 5.13 Check Your Learning

In [24]:
display(-57.06 * 2)
display(180.5 - 114.12)

-114.12

66.38

It is asking for the $\Delta H$ of this process that gives 2 moles, not the enthalpy of formation per mole $\Delta H \rlap \degree _f$.  Therefore, we do not need to first put the overall equation in a form resulting in a single mole of product.
$$ \ce{N2(g) + 2 O2(g) -> 2 NO2(g)} $$
However, we do need to put the constituant reactions in forms that add up to the overall equation.
$$ \ce{N2(g) + O2(g) -> 2NO(g)} \qquad \Delta H = 180.5\ kJ $$
$$ \ce{2NO(g) + O2(g) -> 2NO2(g)} \qquad \Delta H = -114.12\ kJ $$
Summing those gives:
$$ \Delta H = 66.4\ kJ $$

#### Example 5.14 Solution

Find the $\Delta H_{(i)}$ for the top equation using the remaining equations with known $\Delta H$:

$$ \begin{align}
(i)& &&\ce{ClF⁡(g) +F2⁡(g) &&-> ClF3⁢(g)} \qquad
    &&\Delta H \rlap \degree _{(i)} &&= ? \\[1ex]
(ii)& &&\ce{2⁢OF2⁢(g) &&->O2⁡(g) +2⁢F2⁡(g)} \qquad
    &&\Delta H \rlap \degree _{(ii)} &&= -49.4\ kJ \\[1ex]
(iii)& &&\ce{2⁢ClF⁡(g) +O2⁡(g) &&->Cl2O(g) +OF2⁢(g)} \qquad
    &&\Delta H \rlap \degree _{(iii)} &&= + 214.0\ kJ \\
(iv)& &&\ce{ClF3⁢(g) +O2⁡(g) &&->\frac12 ⁢Cl2O(g) + \frac32 ⁢OF2⁢(g)} \qquad
    &&\Delta H \rlap \degree _{(iv)} &&= +236.2\ kJ \\[1ex]
\end{align} $$


In [25]:
display(-49.4 / -2)
display(+214.0 / 2)
display(+236.2 * -1)
display(24.7 + 107.0)
display(131.7 - 236.2)

24.7

107.0

-236.2

131.7

-104.5

To get from reactants to final product, I need:
- Equation (ii) backward and halved.
- Equation (iii) halved.
- Equation (iv) backward.
$$ \begin{alignat}{4}
(i)&\quad&&\ce{ClF⁡(g) +F2⁡(g) &&-> ClF3⁢(g)} \qquad
    &&\Delta H \degree &&= ? \\[1ex]
(ii)& &&\ce{\frac12 O2⁡(g) +⁢F2⁡(g) &&-> ⁢OF2⁢(g)} \qquad
    &&\Delta H \degree &&= +24.7\ kJ \\
(iii)& &&\ce{⁢ClF⁡(g) +\frac12 O2⁡(g) &&->\frac12 Cl2O(g) +\frac12 OF2⁢(g)} \qquad
    &&\Delta H \degree &&= + 107.0\ kJ \\
(iv)& &&\ce{\frac12 ⁢Cl2O(g) + \frac32 ⁢OF2⁢(g) &&->ClF3⁢(g) +O2⁡(g)} \qquad
    &&\Delta H \degree &&= -236.2\ kJ \\[1ex]
\end{alignat} $$

Summing equations (ii) and (iii) gives:
$$ \begin{alignat}{3}
&\ce{ClF⁡(g)  + ⁢F2⁡(g) + O2⁡(g) 
    &&-> ⁢\frac12 Cl2O(g) +\frac32 OF2⁢(g)} \qquad
    &&\Delta H \rlap \degree _{(ii)} &&= +131.7\ kJ \\
\end{alignat} $$

Summing equation (iv) onto the previous result gives:
$$ \begin{alignat}{3}
&\ce{ClF⁡(g) + ⁢F2⁡(g) + O2⁡(g) &&-> ClF3⁢(g) + O2⁡(g)} \qquad
    &&\Delta H \rlap \degree _{(ii)} &&= -104.5\ kJ \\
\end{alignat} $$
Which simplifies to equation (i) and thus gives us our answer:
$$ \begin{alignat}{3}
&\ce{ClF⁡(g) + ⁢F2⁡(g) &&-> ClF3⁢(g)} \qquad
    &&\Delta H \rlap \degree _{(ii)} &&= -104.5\ kJ \\
\end{alignat} $$

#### Example 5.14 Check Your Learning

Calculate $\Delta H \degree$ of equation (i) from the remaining equations:
$$ \begin{alignat}{4}
(i)&\quad&&\ce{2⁢Al⁡(s) +3⁢Cl2⁢(g) && ->2⁢AlCl3⁢(s)} \qquad
    &&\Delta H \degree &&= ? \\
(ii)& &&\ce{HCl⁡(g) &&->HCl⁡(aq)} \qquad
    &&\Delta H \degree &&= -74.8\ kJ \\
(iii)& &&\ce{H2⁡(g) +Cl2⁢(g) &&->2⁢HCl⁡(g)} \qquad
    &&\Delta H \degree &&= -185\ kJ \\
(iv)& &&\ce{AlCl3⁢(aq) &&->AlCl3⁢(s)} \qquad
    &&\Delta H \degree &&= +323\ kJ \\
(v)& &&\ce{2Al⁡(s) +6⁢HCl⁡(aq) &&->2⁢AlCl3⁢(aq) +3⁢H2⁡(g)} \qquad
    &&\Delta H \degree &&= -1049\ kJ \\
\end{alignat} $$

In [26]:
(3 * -185) + (6 * -74.8) + -1049 + (2 * 323)

-1406.8000000000002

To get equation (i) I need to do:
$$ 3 \cdot (iii) + 6 \cdot (ii) + (v) + 2 \cdot (iv) = (i) $$
Therefore:
$$ \Delta H \rlap \degree _{(i)} = -1407\ kJ $$

#### Exercise 5.15 Solution

In [27]:
2 * -207.4 + 90.25 - (3 * 33.2 - 285.83)

-138.32000000000005

Problem statement:
> What is the standard enthalpy change for the reaction:
> $$ \ce{3⁢NO2⁢(g) +H2O(l) -> 2⁢HNO3⁢(aq) +NO⁡(g)} \qquad \Delta H \degree =\ ? $$

From appendix G:
| Substance | Standard Enthalpy of Formation $ \Delta H \rlap \degree _f\ kJ/mol $ |
| - | - |
| $\ce{NO2(g)}$ | 33.2 |
| $\ce{H2O(l)}$ | -285.83 |
| $\ce{HNO3(aq)}$ | -207.4 |
| $\ce{NO(g)}$ | 90.25 |

So
$$ \Delta H \rlap \degree _{\text{reaction}} = -138.3\ kJ $$


In [28]:
display(3 * 33.2)
display(2 * -207.4)
display(-99.6 + 285.83 - 414.8 + 90.25)
display(-99.6 + 285.8 - 414.8 + 90.2)

99.60000000000001

-414.8

-138.32000000000002

-138.39999999999998

#### Exercise 5.15 Check Your Learning

In [29]:
dh_ethanol = -278
dh_H2O = -286
dh_CO2 = -394
dh_O2 = 0
2*dh_CO2 + 3*dh_H2O - (dh_ethanol + 3*dh_O2)

-1368

The combustion formula for ethanol is:
$$ \ce{C2H5OH(l) + 3 O2(g) -> 2 CO2(g) + 3 H2O(l)} $$
The $\Delta H \rlap \degree _c $ is:
$$ \Delta H \rlap \degree _c = -1368\ kJ/mol $$
**Note:** I didn't originally express the answer as per mole.

### Chapter 5 &ndash; Exercises

#### Ex 5.1

While the temperature (an intensive property) of a lit match is high,
the match is small.  Therefore, the total amount of thermal energy it
contains (an extensive property) is small, and the amount of heat flow
it is giving off to you and its environment is small,
making it a poor choice to heat something as big as a person.

#### Ex 5.3

The specific heat (c) of a substance is the amount of heat energy required to change the temperature of 1 gram of the substance by 1 &deg;C.  Since it is per unit mass, it does not depend on the amount of the substance and is an intensive property.

The heat capacity of something is the amount of heat energy required to change the temperature of something by 1 &deg;C.  Since it depends on the amount of the substance, it is an extensive property.

#### Ex 5.5

In [30]:
CNJ = 45.8 * 1.040
CAlJ = 1 / 2.205 * 1000 * 0.897
CAlJexact = 1 * 0.4535924 * 1000 * 0.897
CAlJapprox = 1 / 2.2 * 1000 * 0.897
CAlJgrams = 1 * 464 * 0.897
display(CNJ)
display(CNJ / 4.184)
display(CAlJ)
display(CAlJ / 4.184)
display(CAlJexact)
display(CAlJexact / 4.184)
display(CAlJapprox)
display(CAlJapprox / 4.184)
display(CAlJgrams)
display(CAlJgrams / 4.184)

47.632

11.384321223709367

406.8027210884353

97.22818381654763

406.8723828

97.24483336520076

407.7272727272727

97.44915696158525

416.208

99.47609942638624

Calculate the heat capacity, in joules and in calories per degree, of the following:

(a) 45.8 g of nitrogen gas

(b) 1.00 pound of aluminum metal

$$ \begin{align}
c &:= \frac{J}{g \degree C} \\
C &= m \cdot c \text{ in } \frac{J}{\degree C} \\
C_N &= 47.6\ J/\degree C = 11.4\ cal/\degree C \\[1ex]
C_{Al} &= 407\ J/\degree C = 97.2\ cal/\degree C \\
\end{align} $$

**Note:** Two of the book answers were slightly different, but I'm pretty sure the book is wrong.  I am reporting them as errata.

#### Ex 5.7

In [31]:
qJ = 28.4 * 2.093 * (-1.0 - -23.0)
qc = qJ / 4.184
display(qJ)
display(qc)

1307.7063999999998

312.54933078393873

$$ 1.31 \times 10^3\ J, 313\ cal $$

#### Ex5.9
$$ q = mc\Delta T $$
$$ \Delta T = \frac{q}{mc} $$

In [32]:
14.5 * 10**3 / (485 * 4.184)

7.145532317517889

#### Ex 5.11
$$ q = mc \Delta T $$
$$ c = \frac{q}{m \Delta T} $$

In [33]:
8460 / (437.2 * (68.9 - 19.3))

0.3901292683646666

$$ c_{substance} = 0.390\ J/g \degree C $$
The substance is probably copper.

#### Ex 5.13

Most people find waterbeds uncomfortable unless the water temperature is maintained at about 85 °F. Unless it is heated, a waterbed that contains 892 L of water cools from 85 °F to 72 °F in 24 hours. Estimate the amount of electrical energy required over 24 hours, in kWh, to keep the bed from cooling. Note that 1 kilowatt-hour (kWh) = 3.6 × 10<sup>6</sup> J, and assume that the density of water is 1.0 g/mL (independent of temperature). What other assumptions did you make? How did they affect your calculated result (i.e., were they likely to yield “positive” or “negative” errors)? 

In [34]:
tiC = (85 - 32) / 9 * 5
tfC = (72 - 32) / 9 * 5
dtC = tfC - tiC
display(f'Init: {tiC} C, Final: {tfC} C, Delta: {dtC}')
mbed = 892 * 10**3 * 1.0
Cbed = mbed * 4.184
display(f'm bed: {mbed} g, C bed: {Cbed} J/C')
qJ = Cbed * dtC
qkWh = qJ / (3.6 * 10**6)
display(f'q:{qJ} J, {qkWh} kWh')
onestep = - (((72 - 32) / 9 * 5) - ((85 - 32) / 9 * 5)) * 892 * 10**3 * 1.0 * 4.184 / (3.6 * 10**6)
display(onestep)

'Init: 29.444444444444446 C, Final: 22.22222222222222 C, Delta: -7.222222222222225'

'm bed: 892000.0 g, C bed: 3732128.0 J/C'

'q:-26954257.777777787 J, -7.487293827160497 kWh'

7.487293827160498

$$ \begin{align}
T_{init} &= 85 \degree F = 29.4 \degree C \\
T_{final} &= 72 \degree F = 22.2 \degree C \\
\Delta T &= -7.2 \degree C \\
V_{bed} &= 892\ L \\
m_{bed} &= 892\ L \cdot 1000\ mL/L \cdot 1.0\ g/mL = 8.92 \times 10^5 g \\
c_{water} &= 4.184\ J / g \degree C \\
C_{bed} &= m_{bed} \cdot c_{water} = 3.732128 \times 10^6 \ J/\degree C \\
1\ kWh &= 3.6 \times 10^6\ J \\
q &= C_{bed} \cdot \Delta T = -2.6954258 \times 10^7 J = -7.48729 kWh \\
power &= 7.5 kWh \\
\end{align} $$

I estimate holding the temperature of the waterbed steady at 85 &deg;F for 24 hours will take 7.5 kWh of electrical power.  Assumptions:
- None of the applied heat will go to keep the bed itself warm, just the water.  This would likely result in an underestimate of the needed power.
- Heat will not escape faster to the environment because the water is at a higher temperature over the course of the day.  This would likely result in an underestimate of the needed power.

<strong style="color:red">Note:</strong> The book answer is 7.47 kWh. I find the rounding questionable.  I find the specific answer to three significant digits questionable.   will create an errata.  I forgot to list the assumption of the density of water.

#### Ex 5.15

It depends on the temperature of the external environment.  If the environment was colder than the temperatures inside the calorimeter, such as 20 &deg; C, then a poorer insulator would result in more heat escaping to the outside, reducing the amount of heat measured.  However, if the environment was warmer than the temperatures inside the calorimeter, such as 30 &deg; C, then a poorer insulator would result in heat flowing into the calorimeter, increasng the amount of heat measured.

If we assume that the starting temperature of the reactants, 22 &deg; C is the temperature of the environment, then a poorer insulator would result in more heat escaping to the outside, reducing the amount of heat measured.

#### Ex 5.17

The heat absorbed would appear greater, because taking the heat capacity of the calorimeter itself into account would increase the heat capacity and mass that the measured temperature change was applied to, increasing the measured amount of heat energy transfer.  Another way to look at it is that by taking the calorimeter into account, we would now be accounting for heat that entered the system that had been missed before.

#### Ex 5.19

In [35]:
((0.24 * 45 * 25) + (4.184 * 180 * 95)) / ((4.184 * 180) + (0.24 * 45))

94.01036757775682

Reaching back to the equation I already munged around for Example 5.3, we have.
$$ \begin{align}
T_{final} &=
\frac{c_b \cdot m_b \cdot T_{b\ initial} + c_a \cdot m_a \cdot T_{a\ initial}}
     {c_a \cdot m_a + c_b \cdot m_b}
\end{align} $$

If we call $a$ the coffee and $b$ the spoon, then...
$$ \begin{align}
T_{a,init} &= 95 \degree C \\
T_{b,init} &= 25 \degree C \\
c_a &= 4.184\ J/g \degree C \text{ assumed same as water} \\
c_b &= 0.24\ J/g \degree C \\
m_a &= 180\ g \\
m_b &= 45\ g \\
T_{final} &= 94 \degree C \\
\end{align} $$

So the coffee temperature dropped by 1 &deg; C.

#### Ex 5.21
The temperature of the cooling water as it leaves the hot engine of an automobile is 240 °F. After it passes through the radiator it has a temperature of 175 °F. Calculate the amount of heat transferred from the engine to the surroundings by one gallon of water with a specific heat of 4.184 J/g °C.

In [36]:
tiC = (240 - 32) * 5 / 9
tfC = (175 - 32) * 5 / 9
dt = tfC - tiC
display(f'T init: {tiC} C, T fin: {tfC} C, delta T: {dt}')
quarts_per_gallon = 4
quarts_per_liter = 1.057
liters_per_quart = 0.9463
ml_per_liter = 1000
g_per_ml = 1.0
vwg = 1
vwl = 1 * quarts_per_gallon / quarts_per_liter
vwl2 = 1 * quarts_per_gallon * liters_per_quart
vwml = vwl * ml_per_liter
vwml2 = vwl2 * ml_per_liter
display(f'Vol water: {vwg} gal, {vwl} or {vwl2} L, {vwml2} mL')
mwg = vwml * g_per_ml
mwg2 = vwml2 * g_per_ml
display(f'Mass water: {mwg} g or {mwg2} g')
q = mwg * 4.184 * dt
q2 = mwg2 * 4.184 * dt
display(f'q: {q} J or {q2} J')
display(f'Heat transferred to surroundings = {-round(q2 / 1000, 0)} kJ')

'T init: 115.55555555555556 C, T fin: 79.44444444444444 C, delta T: -36.111111111111114'

'Vol water: 1 gal, 3.784295175023652 or 3.7852 L, 3785.2000000000003 mL'

'Mass water: 3784.295175023652 g or 3785.2000000000003 g'

'q: -571764.9532219069 J or -571901.6622222223 J'

'Heat transferred to surroundings = 572.0 kJ'

**Note:** Book rounded it to 5.7 x 10<sup>2</sup> kJ.

#### Ex 5.23
If a reaction produces 1.506 kJ of heat, which is trapped in 30.0 g of water initially at 26.5 °C in a calorimeter like that in Figure 5.12, what is the resulting temperature of the water?

In [37]:
tf = 1.506 * 1000 / (30.0 * 4.184) + 26.5
tf

38.4980879541109

$$ \begin{align}
q &= mc \Delta T = mc(T_{final} - T_{init}) \\[1ex]
\Delta T &= \frac{q}{mc} \\[1ex]
T_{final} - T_{init} &= \frac{q}{mc} \\[1ex]
T_{final} &= \frac{q}{mc} + T_{init} \\[1ex]
T_{final} &= 38.5 \degree C \\[1ex]
\end{align} $$

#### Ex 5.25
Dissolving 3.0 g of CaCl<sub>2</sub>(s) in 150.0 g of water in a calorimeter (Figure 5.12) at 22.4 °C causes the temperature to rise to 25.8 °C. What is the approximate amount of heat involved in the dissolution, assuming the specific heat of the resulting solution is 4.18 J/g °C? Is the reaction exothermic or endothermic?

In [38]:
display((150.0 + 3.0) * 4.18 * (25.8 - 22.4))
display((150.0) * 4.18 * (25.8 - 22.4))

2174.436000000001

2131.8000000000015

The amount of heat involved is approximately 2.17 kJ.  The reaction is exothermic. 

**Note:** The book gave -2.2 kJ and exothermic.

#### Ex 5.27

In [39]:
(100 + 3.15 + 1.52) * 4.20 * 3.1

1362.8034000000002

Approximately 1.4 kJ of heat were absorbed.

#### Ex 5.29

If the 3.21 g of NH<sub>4</sub>NO<sub>3</sub> in Example 5.6 were dissolved in 100.0 g of water under the same conditions, how much would the temperature change? Explain your answer.  In Example 5.6, it is dissolved in 50.0 g of water instead and the temperature drops from 24.9&deg;C to 20.3&deg;C, a drop of 4.6&deg;C, absorbing 1.0 kJ of energy.

In [55]:
display(24.9 - 20.3)
q_example_5_6 = -((3.21 + 50.0) * 4.184 * (20.3 - 24.9))
q_rounded_5_6 = 1000
q_book_intermediate_5_6 = 4.184 * 53.2 * -4.6
display(f'q from 5.6: {q_example_5_6}, rounded: {q_rounded_5_6}')
dt = q_example_5_6 / ((100.0 + 3.21) * 4.184)
dt_from_rounded = q_rounded_5_6 / ((100.0 + 3.21) * 4.184)
dt_from_intermediate = q_book_intermediate_5_6 / ((100.0 + 3.21) * 4.184)
display(f'dT: {dt}, from round: {dt_from_rounded}, from inter: {dt_from_intermediate}')
display(f'T final: {24.9 - dt}, T final from round: {24.9 - dt_from_rounded}')
display(f'Short T final 1: {24.9 - ((24.9 - 20.3) / 2)}')
display(f'Short T final 2: {24.9 - (1.0 * 1000 / ((100.0 + 3.21) * 4.184))}')
display(f'Short q: {-(50.0 + 3.21) * 4.184 * (20.3 - 24.9)}')
display(f'Short T final 3: {24.9 - (1024.1 / ((100.0 + 3.21) * 4.184))}')

4.599999999999998

'q from 5.6: 1024.1009439999996, rounded: 1000'

'dT: 2.3715337661079343, from round: 2.3157226638665565, from inter: -2.371088072861157'

'T final: 22.528466233892065, T final from round: 22.584277336133443'

'Short T final 1: 22.6'

'Short T final 2: 22.584277336133443'

'Short q: 1024.1009439999996'

'Short T final 3: 22.528468419934256'

Assuming:
- Solution specific heat that of water (like example 5.6 assumed)
- No heat transferred with environment
- NH<sub>4</sub>NO<sub>3</sub> was the limiting reactant so that energy absorbed is same
Then...
$$ \begin{align}
q_{rx} &= -q_{sol} = -mc \Delta T \\[1ex]
\Delta T &= \frac{q_{sol}}{mc} \\
    &= \frac{1000\ J}{(100.0\ g + 3.21\ g) \cdot 4.184\ J/g \degree C} \\[1ex]
    &= 2.3\ \degree C \\
    &\  \mathbf{or} \\
    &= \frac{1024.1009439999996\ J}{(100.0\ g + 3.21\ g) \cdot 4.184\ J/g \degree C} \\[1ex]
    &= 2.4\ \degree C \\
\end{align} $$
In other words, I worked backwards from the heat absorbed and essentially used the heat capacity of the combined increased solvent and unchanged solute to calculate a new temperature change.  The result makes sense.  The majority of the mass was in the water, so doubling the water should approximately halve the temperature change.

<strong style="color:red">Note:</strong> The book answer was 22.6.  I made 3 errata.

#### Ex 3.31
When a 0.740-g sample of trinitrotoluene (TNT), $\ce{C7H5N2O6}$, is burned in a bomb calorimeter, the temperature increases from 23.4 °C to 26.9 °C. The heat capacity of the calorimeter is 534 J/°C, and it contains 675 mL of water. How much heat was produced by the combustion of the TNT sample?

In [79]:
C_sur = 534 + 675 * 1.0 * 4.184
q_rx = C_sur * (26.9 - 23.4)
display(f'C surrounding: {C_sur}, q rx: {q_rx}')
C_total = 534 + (675 + .740) * 4.184
q_total = C_total * (26.9 - 23.4)
display(f'q including bad approximation of c of TNT itself: {q_total}')
display('Ok, now try to explain the book answer with intermediate rounding.')
C_water = 675 * 4.184
C_water_r = round(C_water, -1)
C_all = C_water_r + 534
C_all_r = round(C_all, -1)
display(f'C water: {C_water} rounds to {C_water_r}, C all: {C_all} rounds to {C_all_r}')
dt = 26.9 - 23.4
display(f'Delta t: {dt}')
q_all = C_all_r * dt
q_all_r = round(q_all, -2)
display(f'q all: {q_all} rounds to {q_all_r}')
display(f'one step: {(534 + (675 * 1.0 * 4.184)) * (26.9 - 23.4)}')

'C surrounding: 3358.2000000000003, q rx: 11753.7'

'q including bad approximation of c of TNT itself: 11764.53656'

'Ok, now try to explain the book answer with intermediate rounding.'

'C water: 2824.2000000000003 rounds to 2820.0, C all: 3354.0 rounds to 3350.0'

'Delta t: 3.5'

'q all: 11725.0 rounds to 11700.0'

'one step: 11753.7'

$$ \begin{align}
C_{surroundings} &= C_{calorimeter} + C_{water} \\
    &= 534\ J/\degree C + 675\ mL \cdot 1.0\ g/mL \cdot 4.184\ J/g \degree C \\
    &= 3358.2\ J/\degree C \\
q_{rx} &= -q_{surroundings} = -C_{surroundings} * \Delta T \\
    &= -3358.2\ J/\degree C \cdot (26.9 - 23.4 \degree C) \\
    &= -11.8\ kJ
\end{align} $$
So 11.8 kJ of heat was produced, or 12 kJ if you want to round based on the significant figures in the 1.0 g/mL approximation for the density of water.

<strong style="color:red">Note:</strong> The book answer, 11.7 kJ, appears to have used intermediate rounding, possibly without even guard digits.  I added an errata.

#### Ex 5.33
The amount of fat recommended for someone with a daily diet of 2000 Calories is 65 g. What percent of the calories in this diet would be supplied by this amount of fat if the average number of Calories for fat is
9.1 Calories/g? 

In [81]:
(65 * 9.1) / 2000 * 100

29.575000000000003

Looks like 30%.

#### Ex 5.35
What is the maximum mass of carbohydrate in a 6-oz serving of diet soda that contains less than 1 Calorie per can if the average number of Calories for carbohydrates is 4.1 Calories/g?

In [83]:
1 / 4.1

0.24390243902439027

Less than .24 g (or .2 g if 1 Calorie significant figures is used.)

#### Ex 5.37
A serving of a breakfast cereal contains 3 g of protein, 18 g of carbohydrates, and 6 g of fat. What is the Calorie content of a serving of this cereal if the average number of Calories for fat is 9.1 Calories/g, for carbohydrates is 4.1 Calories/g, and for protein is 4.1 Calories/g? 

In [84]:
3 * 4.1 + 18 * 4.1 + 6 * 9.1

140.7

140 Calories.

#### Ex 5.39
Explain how the heat measured in Example 5.5 differs from the enthalpy change for the exothermic reaction described by the following equation:

$$\ce{HCl⁡(aq) + NaOH⁡(aq) -> NaCl⁡(aq) + H2O(l)}$$

(Note that the chemical formula is identical to the Example 5.5 coffee cup calorimeter example, including the (aq) and (l) states of matter.)

My answer:

Numerically they would only differ in that the enthalpy would include any heat lost to the environment, including any heat lost to the coffee cup calorimeter itself.  In other words, the definition enthalpy change does not make use of the measuring assumptions used in Example 5.5.  Other than that, they would be numerically identical.

Where they differ is in the meaning.  The enthalpy change only equals the heat quantity under the assumption of constant pressure so that the work term of the enthalpy definition is cancelled out by the P times &Delta;V work done by or on the system when it is under constant pressure but not constant volume.

<strong style="color:blue">Oops.</strong> The book answer is:
> The enthalpy change of the indicated reaction is for exactly 1 mol HCL and 1 mol NaOH; the heat in the example is produced by 0.0500 mol HCl and 0.0500 mol NaOH.

From book Ch 5.3:
> This &Delta;H value indicates the amount of heat associated with the reaction involving the number of moles of reactants and products as shown in the chemical equation.

That Ch 5.3 book quote was from the section on $\Delta H$, before any of the sections on $\Delta H \degree$, $\Delta H \rlap \degree _c$, or $\Delta H \rlap \degree _f$.